### ISSUE TITLE
How i can modify ManifestData before deploying?

### ISSUE THREAD

**Issue by mostafaashraf92** [rokucommunity/roku-deploy#43]
How i can modify ManifestData before deploying? i already know how to deal with rokudeploy.json but where i can modify manifest data ? As example i want to change the title of the app

**Comment by triwav** [comment]
Are you wanting to do this as part of the callback while deploying or before hand?

**Comment by mostafaashraf92** [comment]
@triwav Yes as a part of the callback. I want to automate it , i wonder if it is done the same way as rokudeploy.json ?  Where i can write this part?

**Comment by mostafaashraf92** [comment]
@triwav I can see that when i write in rokudeploy.json the options is provided to deployment, but i can see info object can be passed as well , so where i can instantiate this info object ?

**Comment by triwav** [comment]
Got it. So you would take the `ManifestData` and modify it out how you want. In your case that should be something like `manifestData.title = 'New Title'`. Then you need to change it back to a string like `const stringifiedManifest = rokuDeploy.stringifyManifest(manifestData);` and then write that to the manifest in your staging folder like
```
const manifestPath = path.join(info.stagingFolderPath, 'manifest');
await fsExtra.writeFile(manifestPath, stringifiedManifest);
```
